sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample4_ReadRevisionHistory.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.Data.AppConfiguration.Samples
+{
+    [LiveOnly]
+    public partial class ConfigurationSamples
+    {
+        [Test]
+        /*
+         * This sample demonstrates how to read the revision history of one or
+         * more Configuration Settings in a Configuration Store.
+         *
+         * To do this, we will create two settings, change them twice to create
+         * revisions, then read the revision history first of one of them by
+         * itself, and then the revision history of both of them together.
+         */
+        public async Task ReadRevisionHistory()
+        {
+            // Retrieve the connection string from the configuration store.
+            // You can get the string from your Azure portal.
+            var connectionString = Environment.GetEnvironmentVariable("APPCONFIGURATION_CONNECTION_STRING");
+
+            // Instantiate a client that will be used to call the service.
+            var client = new ConfigurationClient(connectionString);
+
+            // Create the Configuration Settings to be stored in the Configuration Store.
+            var productionEndpoint = new ConfigurationSetting("endpoint", "https://production.endpoint.com", "production");
```

Comment:
Do your other samples also use this same name? If so it might make your "GetRevisionsAsync" below dump out some unrelated revisions.